sklearn/metrics/tests/test_common.py

```diff
@@ -1247,3 +1251,26 @@ def test_thresholded_multilabel_multioutput_permutations_invariance(name):
 
         current_score = metric(y_true_perm, y_score_perm)
         assert_almost_equal(score, current_score)
+
+
+@pytest.mark.parametrize(
+    'name',
+    set(THRESHOLDED_METRICS) - METRIC_UNDEFINED_BINARY_MULTICLASS)
+def test_multiclass_score_permutation_invariance(name):
+    n_samples, n_classes = 100, 3
+    random_state = check_random_state(0)
+
+    y_score = random_state.rand(n_samples, n_classes)
+    y_score = y_score / y_score.sum(axis=1, keepdims=True)
+    y_true = random_state.randint(0, n_classes, size=n_samples)
+
+    metric = ALL_METRICS[name]
+    score = metric(y_true, y_score)
+    for perm in permutations(range(n_classes), n_classes):
+        inv_perm = np.zeros(n_classes, dtype=int)
```

Comment:
Why do you need an inverse permutation here?

Reply:
For example, lets say `perm=(1, 2, 0)`. This maps 0 to 1, 1 to 2, and 2 to 0. Lets say `y_true=np.array([2,1,0,2,1])`, the permutation can be calculated: `np.take(perm, y_true)`, which results in `y_true_perm=array([0, 2, 1, 0, 2])`. To adjust `y_score`, we need the inverse permutation, `array([2, 0, 1])`, to map columns 2 to 0, 0 to 1, and 1 to 2, using numpy indexing.